spec/models/comment_spec.rb

```diff
@@ -82,4 +87,27 @@
       end
     end
   end
+
+  describe "#mentioned audiences" do
+    around(:each) do |example|
+      I18n.with_locale(['de', 'fe', 'en'].sample) do
+        example.run
+      end
+    end
+
+    it "mentioning an audience should return corresponding audience" do
+      audience = Audience.all.sample
+      audience_translated = Audience.send(audience).translate
+
+      comment1 = create :comment, body: "Hey, @#{audience_translated}"
+      comment2 = create :comment,
+                        body: "<p>Hey, <span class=\"mention\"
+                          data-mention-id=\"#{audience_translated}\"
```

Comment:
oh, this part should be "group" or "discussion" no matter what language. So you only need to translate the user generated content.